src/app/modules/account/pages/details/components/contact-information/contact-information.component.html

```diff
@@ -0,0 +1,31 @@
+<ion-card>
+  <ion-card-header>
+    <ion-card-title>Contact information</ion-card-title>
+  </ion-card-header>
+  <ion-card-content>
+    <ion-grid>
+      <ion-row>
+        <ion-col size="6">
+          <div>
+            <h2>Phone</h2>
+            <p>{{ getPhoneNumber() }}</p>
+          </div>
+          <div>
+            <h2>Email address</h2>
+            <p>{{ account?.email }}</p>
```

Comment:
I think for this we want to use the `account?.contactInformation?.emails` array.  you can probably just display the first email in the array or iterate over them.
```
interface ContactInformation {
  privacy?: "public" | "private" | "specific-users"; // Privacy setting
  addresses?: Nullable<Address>[];
  phoneNumbers: PhoneNumber[];
  emails: Email[];
  preferredMethodOfContact: "Email" | "Phone" | "SMS" | "Fax";
}
```